SmartContract/src/errors.txt

```diff
@@ -1,7 +1,7 @@
 E1      Specified agreement has no jury
```

Comment:
is there no way to formalize these errors? As in making a formal document that will stay on the repo?